### PR TITLE
Release v2.6.2

### PR DESCRIPTION
**Pull request by niik**
## Description

Looking for the PR for the upcoming v2.6.2 production release? Well you've just found it, congratulations! :tada:

We're targeting a release later this week

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

**Review by sergiou87 — DISMISSED** [review]
LGTM!

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.6.2": [
+      "[Improved] Upgrade embedded Git LFS to 2.13.2 - #11394",
+      "[Fixed] Addressed an issue where pull requests could fail to update until the user switched repositories - #11241",
+      "[Fixed] Fork behavior changes are now reflected in the app immediately - #11327",
+      "[Fixed] Checking out a pull request from a fork for the first time now correctly identifies the remote branch to create the branch from - #11267",
+      "[Fixed] Don't offer to stash or move changes when clicking on the currently checked out branch in the branch list - #11235",
+      "[Fixed] Forked repository remotes are no longer removed when there's local branches tracking them - #11266",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Forked repository remotes are no longer removed when there are local branches tracking them - #11266",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.6.2": [
+      "[Improved] Upgrade embedded Git LFS to 2.13.2 - #11394",
+      "[Fixed] Addressed an issue where pull requests could fail to update until the user switched repositories - #11241",
+      "[Fixed] Fork behavior changes are now reflected in the app immediately - #11327",
+      "[Fixed] Checking out a pull request from a fork for the first time now correctly identifies the remote branch to create the branch from - #11267",
+      "[Fixed] Don't offer to stash or move changes when clicking on the currently checked out branch in the branch list - #11235",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Don't offer to stash changes when clicking on the currently checked out branch in the branch list - #11235",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.6.2": [
+      "[Improved] Upgrade embedded Git LFS to 2.13.2 - #11394",
+      "[Fixed] Addressed an issue where pull requests could fail to update until the user switched repositories - #11241",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Pull requests no longer fail to update until the user switches repositories - #11241",
```

**Review by billygriffin — COMMENTED** [review]
Minor changes

**Review by billygriffin — DISMISSED** [review]

**Review by sergiou87 — DISMISSED** [review]
LGTM!

**Review by sergiou87 — APPROVED** [review]
Looking good! :shipit:

**Review by aleee1333 — COMMENTED** [review]
Ok

**Comment by arbadebaybay** [comment]
arbadebay@gmail.com

**Comment by arbadebaybay** [comment]
sriudom2523++Bb

**Comment by arbadebaybay** [comment]
รวม

**Pull request merged**